### PR TITLE
update dependencies from npm

### PR DESCRIPTION
**Pull request by viztor**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
     "jquery.easing": "^1.4.1",
     "jquery.transit": "^0.9.12",
     "katex": "^0.10.2",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
```
Comment (iceb0y) [inline package.json]: Why is only `loadsh` updated in `package.json`, but a lot of packages get updated in `package-lock.json`?

Comment (viztor) [reply package.json]: > Why is only `loadsh` updated in `package.json`, but a lot of packages get updated in `package-lock.json`?

these are the dependencies needed by lodash as they update it.

Comment (moesoha) [reply package.json]: Looks like you also upgraded `babel` in lock file.

**Comment by moesoha** [comment]
This branch is outdated. Thanks for your contribution.

**Pull request closed without merging**